express_relay/sdk/js/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@pythnetwork/express-relay-evm-js",
-  "version": "0.8.1",
+  "name": "@pythnetwork/express-relay-js",
+  "version": "0.1.0",
```

Comment:
not sure what the best practice is, but I'd rather keep incrementing the versions